skinfo/metrics.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 
 
-def entropy(x, bins):
+def entropy(x, bins, normalize=False, probabilities=False):
```

Comment:
I would rather go for `use_probs` or `as_probs`, because `probabilities` is quite long and at least for me not that clear.